src/components/ReactTags.tsx

```diff
@@ -107,15 +119,22 @@ const ReactTags = (props: ReactTagsProps) => {
         '[Deprecated] autofocus prop will be removed in 7.x so please migrate to autoFocus prop.'
       );
     }
-    
+
     if ((autofocus || (autoFocus && autofocus !== false)) && !readOnly) {
       resetAndFocusInput();
     }
   }, [autoFocus, autoFocus, readOnly]);
 
   useEffect(() => {
     updateSuggestions();
-  }, [query, props.suggestions]);
+  }, [query, props.suggestions, props.enableFuzzySearch]);
```

Comment:
do we need `enableFuzzySearch` here ?

Reply:
yep

Reply:
can you please elaborate on why ? Since there is already a fuzzy search memo which is running for the same so why do we still need to run this when option is updated?

Reply:
This I added when I was writing test cases. Upon re-rendering, react tags with enabling fuzzy-search, suggestions were not getting updated. 

By making `props.enableFuzzySearch` dependency of useEffect, I am trying to update the suggestions, based on fuzzySearch, rather the previous one . 

This might not be real-world case, where you update the `enableFuzzySearch` prop in between. Let me know what's your opinion ?

Reply:
Hmm, I see, alright let it be for now.